public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -884,6 +885,41 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     });
   }
 
+  public getDashboardPanels = () => {
+    if (!(this.state.body instanceof SceneGridLayout)) {
+      throw new Error('Dashboard scene layout is not SceneGridLayout');
+    }
+
+    const sceneGridLayout = this.state.body;
+    const gridItems = sceneGridLayout.state.children;
+    if (!gridItems) {
+      return [];
+    }
+    return gridItems.map((vizPanel) => {
+      if (!(vizPanel instanceof DashboardGridItem)) {
+        throw new Error('Trying to get a panel in a layout that is not DashboardGridItem');
+      }
+      return vizPanel.state.body;
+    });
+  };
+
+  public hasDashboardAngularPlugins = () => {
```

Comment:
why an instance / arrow function? Always try to use prototype functions as a default

Reply:
no reason behind this, tbh muscle memory without much thinking. I will keep this in mind, thanks for the suggestion